### PR TITLE
data: VarchasvH

### PR DESCRIPTION
**Pull request by VarchasvH**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4897"></a>

### CODE REVIEW

**Comment by shubhsharma19** [comment]
LGTM 👍

File: data/VarchasvH.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name" : "Varchasv Hoon",
+  "displayStaticPublic" : true,
```
Comment (imbeshat) [inline data/VarchasvH.json:3]:
```suggestion
```

File: data/VarchasvH.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "type" : "personal"
```
Comment (imbeshat) [inline data/VarchasvH.json:4]:
```suggestion
  "type" : "personal",
```

File: data/VarchasvH.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "bio" : "DevOps Engineer | Open Source Enthusiast | Constantly learning and Constantly Delivering",
+  "avatar" : "https://github.com/VarchasvH.png",
```
Comment (imbeshat) [inline data/VarchasvH.json:6]:
```suggestion
```

File: data/VarchasvH.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "links" : [
+    {
+      "name" : "Follow me on Twitter",
+      "url" : "https://twitter.com/VarchasvH/",
+      "icon" : "FaTwitter"
+    },
+
+    {
+      "name" : "Catch me Online here"
```
Comment (imbeshat) [inline data/VarchasvH.json:15]:
```suggestion
      "name" : "Catch me Online here",
```

File: data/VarchasvH.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      "url" : "https://github.com/VarchasvH",
+      "icon" : "FaGithub"
+     },
+
+    {
+      "name" : "See what I am learning"
```
Comment (imbeshat) [inline data/VarchasvH.json:21]:
```suggestion
      "name" : "See what I am learning",
```

File: data/VarchasvH.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      "url" : "https://varchasvh.hashnode.dev"
```
Comment (imbeshat) [inline data/VarchasvH.json:22]:
```suggestion
      "url" : "https://varchasvh.hashnode.dev",
```

File: data/VarchasvH.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+      "icon" : "SiHashnode"
+    },
+  
+    {
+      "name" : "My Portfolio"
```
Comment (imbeshat) [inline data/VarchasvH.json:27]:
```suggestion
      "name" : "My Portfolio",
```

File: data/VarchasvH.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      "url" : "https://varchasvh.carrd.co"
```
Comment (imbeshat) [inline data/VarchasvH.json:28]:
```suggestion
      "url" : "https://varchasvh.carrd.co",
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please, update these.

File: data/VarchasvH.json (hunk 1 above)
Comment (shubhsharma19) [reply data/VarchasvH.json:3]: It looks fine to me

**Comment by VarchasvH** [comment]
> Please, update these.

Done!

File: data/VarchasvH.json (hunk 1 above)
Comment (imbeshat) [reply data/VarchasvH.json]: @shubhsharma19, please see this #4652

**Review by imbeshat — APPROVED** [review]

**Comment by shubhsharma19** [comment]
Looks good 👍

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**